sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -852,7 +852,20 @@ void gnrc_ipv6_netif_init_by_dev(void)
             _add_addr_to_entry(ipv6_if, &addr, 64, 0);
 
         }
-
+#ifdef GNRC_IPV6_STATIC_LLADDR
+        /* parse addr from string and explicitely set a link lokal prefix
+         * if ifnum > 1 each interface will get its own link local address
+         * with GNRC_IPV6_STATIC_LLADDR + i
+         */
+        #define STR(x) #x
```

Comment:
I don't think it is handy... it just hides pre-processor magic which we try to avoid (also the reason why we use inline functions instead of macros). Also: are you sure this is what you want to do? Afaik this makes the input a string, so `lladdr_str` below becomes `"GNRC_IPV6_STATIC_LLADDR"`.